contracts/wrapper.sol

```diff
@@ -406,7 +406,7 @@ contract WrapAndUnWrap is OwnableUpgradeable {
             uniswapExchange.swapExactETHForTokens{value:amount}(0, addresses, address(this), 1000000000000000 );
```

Comment:
Please pass the min amount calculated from `userSlippageTolerance` as the first param in this method, because this is what will cause the function to revert in case of high slippage. Please [see](https://uniswap.org/docs/v2/smart-contracts/router02/#swapexactethfortokens)